types/src/ledger_info.rs

```diff
@@ -132,6 +132,10 @@ impl LedgerInfo {
         self.consensus_data_hash = consensus_data_hash;
     }
 
+    pub fn set_consensus_block_id(&mut self, consensus_block_id: HashValue) {
```

Comment:
#[cfg(test)]

Reply:
do you mean I should add #[cfg(test)] signature to this function? or add a unit test under #[cfg(test)]?

Reply:
we should add #[cfg(test)] for function only used for tests

Reply:
`set_consensus_block_id()` is being called in `placeholder_certificate_for_block()` in `test_utils.rs`, so I'm not able to add `#[cfg(test)]`. I'll leave it out for now, but please let me know if this will be a problem.

Reply:
you probably need `#[cfg(any(test, feature = "fuzzing"))]` since this is what `test_utils.rs` has